static/app/components/events/interfaces/frame/deprecatedLine.tsx

```diff
@@ -424,6 +424,8 @@ export class DeprecatedLine extends Component<Props, State> {
                           sourceResolutionResults={
                             this.props.frameSourceResolutionResults!
                           }
+                          orgSlug={this.props.organization?.slug}
+                          projectId={this.props.event.projectID}
```

Comment:
I could have reused the `orgSlug` and `project` present in `analyticsParams` prop, but I prefer it to be explicit